### PR TITLE
Update 00-using-this-tutorial.md

### PR DESCRIPTION
**Pull request by kishankarun**
Correction in the link regarding the whl file.

I think the link given was wrong? Did you mean piwheels? https://www.piwheels.org/project/puepy/

### CODE REVIEW

File: docs/tutorial/00-using-this-tutorial.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Each of the examples in this tutorial can be run on PyScript.com, a web environment that lets you write, test, and share PyScript code. Alternatively, you can clone the [PuePy git repo](https://github.com/kkinder/puepy) and run a live web server with each example included.
 
-The PyScript.com environment uses the PuePy `.whl` file, as [downloadable from PyPi](https://github.com/kkinder/puepy), while the examples in the git repository are served with PuePy directly from source files on disk.
+The PyScript.com environment uses the PuePy `.whl` file, as [downloadable from PiWheels](https://www.piwheels.org/project/puepy/), while the examples in the git repository are served with PuePy directly from source files on disk.
```
Comment (kkinder) [inline docs/tutorial/00-using-this-tutorial.md]: I'm not familiar with piwheels.org. What is the advantage it has over PyPi: https://pypi.org/project/puepy/#files?

**Comment by kkinder** [comment]
Looks fine, though I'm wondering why we'd switch to piwheels over just pypi?

File: docs/tutorial/00-using-this-tutorial.md (hunk 1 above)
Comment (kishankarun) [reply docs/tutorial/00-using-this-tutorial.md]: Frankly I am also not familiar with pywheels.

Actually, I meant that the URL you had mentioned 

```
[downloadable from PyPi]
```

Was pointing to

```
(https://github.com/kkinder/puepy)
```

I couldn't find the .whl file from the github.com link. 
The link you meant was https://pypi.org/project/puepy/#files right?

Comment (kkinder) [reply docs/tutorial/00-using-this-tutorial.md]: Ah, I see. Yes, that link should be fixed. Good catch.

**Review by kkinder — APPROVED** [review]

**Comment by kkinder** [comment]
Merged! Thanks for contributing.

**Comment by kishankarun** [comment]
> Looks fine, though I'm wondering why we'd switch to piwheels over just pypi?

Sorry for the confusion caused. 
I was searching the web to find the puepy.whl file instead of checking the pypi site of puepy, where it was attached...

**Pull request merged**